modules/geocoder_field/src/Plugin/Geocoder/Field/DefaultField.php

```diff
@@ -127,11 +128,12 @@ public function getSettingsForm(FieldConfigInterface $field, array $form, FormSt
         'action' => 'order',
         'relationship' => 'sibling',
         'group' => 'plugins-order-weight',
-      ]],
+      ],
+      ],
```

Comment:
If we open with inline `[[`, we should close in the same way. Otherwise let's move the opening to 2 lines and indent properly.